site/_config.yml

```
@@ -27,6 +27,7 @@ remote_theme: "mmistakes/jekyll-theme-basically-basic@1.4.5"
 
 lang: en-US
 title: MAKE Roanoke
+tag: Where people come to make stuff
```

I get the sentiment here, but "make stuff" is too generic for the first text you see on the front page. Stuff might include "tractors", "homes", and "popcorn".  I'm pretty sure we aren't making those.

```suggestion
tag: A maker initiative in Roanoke, Virginia.
```

---

Fine with me.  I just hate the empty space.  I also want to change the variable from site.tag to site.tagline so that the term tag does not get too overloaded.